force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -270,18 +333,28 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Book> generateBookList(Book myBook1) {
         // Create a list of books
+        Book book1 = new Book();
+        Book book2 = new Book();
 
         // Update the title and author of myBook1
-
         // Update the pages for myBook1 to 352
+        book1.title = 'Harry Potter and the Chamber of Secrets';
+        book1.author = 'J.K. Rowling';
+        book1.pages = 352;
+       
 
         // Create a new Book object with title "A Brief History of Time" and author "Stephen Hawking"
-
         // Update the pages for A Brief History of Time to 256
+        book2.title = 'A Brief History of Time';
+        book2.author = 'Stephen Hawking';
+        book2.pages = 256;
 
         // Add the book to the list
-        
-        return null; // Replace null with the variable you used to store the result      
+        List<Book> bookList = new List<Book>();
+        bookList.add(book1);
+        bookList.add(book2);
+
+        return bookList; // Replace null with the variable you used to store the result      
```

Comment:
If you had not hard coded your constructor you have have saved a lot  of work here. 
```
 public static List<Book> generateBookList(Book myBook1) {
        // Create a list of books
        List<Book> books = new List<Book>();
        myBook1.author = 'J.K. Rowling';
        myBook1.title = 'Harry Potter and the Chamber of Secrets';
        myBook1.pages = 352;
        books.add(myBook1);
        books.add(new Book('A Brief History of Time', 'Stephen Hawking', 256));

        return books;

        // Update the title and author of myBook1

        // Update the pages for myBook1 to 352

        // Create a new Book object with title "A Brief History of Time" and author "Stephen Hawking"

        // Update the pages for A Brief History of Time to 256

        // Add the book to the list
    }
````